app/assets/src/components/ui/controls/category_search_box.scss

```diff
@@ -0,0 +1,45 @@
+@import "~styles/themes/colors";
+
+:global(.prompt) {
```

Comment:
Can you encapsulate all this in a unique class that you assign to your main component?

Reply:
yes, thanks